concent_api/core/utils.py

```diff
@@ -0,0 +1,65 @@
+import datetime
```

Comment:
File name: `core/utils.py`

Could you maybe call it `timing_helpers.py`? I think we should try to avoid putting all "utils" on one big heap. Instead, try to group them in some logical way.

Reply:
Then I would put them in `utils\timing_helper` :) I think that we should follow django convention on naming files inside apps. Currently in `core` we have `message_handlers`, `subtask_helper` and `transfers_operations` and I never remember where to look for a given function, because current distribution of them is not intuitive. I would really like to avoid that. If I see `utils.py` I know what to expect inside. If they grow to much, we can split them into `core\utils\type_1.py`, `core\utils\type_2.py` etc.

Reply:
So maybe `core/helpers/timing.py`? The `utils` part is the one I have issues with. To me `utils` suggests general-purpose utilities, not something intimately tied to the business logic. `helpers` is a tad better though I'd really prefer a more specific name. I think that the best organization is when every class has its own file though when they're small it sometimes makes sense to group several together.

But in general, if you don't like the current organization then I think you should meet with @dybi and figure out a better one. There was a moment where we started splitting files and I was really assuming that you both have it under control. That you have some idea and we're heading towards it. I think that it's something that we need to plan ahead rather than assume that it will just happen spontaneously.